# author: Denis Kristak (INUITS)
# set correct dependecy versions && expect *.a files to copy, rather than *.so
diff -ruN GenomeWorks-dev_orig/pygenomeworks/requirements.txt GenomeWorks-dev/pygenomeworks/requirements.txt
--- GenomeWorks-dev_orig/pygenomeworks/requirements.txt	2021-06-08 19:42:06.199086912 +0200
+++ GenomeWorks-dev/pygenomeworks/requirements.txt	2021-06-08 19:41:42.299168164 +0200
@@ -16,12 +16,12 @@
 
 
 
-Cython~=0.29.14
+Cython==0.29.21
 intervaltree==3.1.0
-matplotlib==3.3.2
-networkx==2.4
-numpy~=1.17.5
-pytest==4.4.1
+matplotlib==3.3.3
+networkx==2.5.1
+numpy==1.19.4
+pytest==6.1.1
 setuptools>=41.4.0
-sortedcollections==1.1.2
-tqdm~=4.31
+sortedcollections==2.1.0
+tqdm==4.56.2
